reference-architecture/gce-cli/ansible/playbooks/roles/openshift-ansible-installer/tasks/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: download openshift ansible installer from github
+  git:
+    repo: https://github.com/openshift/openshift-ansible.git
+    dest: '{{ playbook_dir }}/../../../../../openshift-ansible'
```

Comment:
I've considered this risk, but looking at the history of the release branches it's clear that the development is done in the `master` branch and the changes in the `release` branches are usually quickly followed by a release tag (and the rpm). So the risk is there, but quite low I would say. Main reason why I'm doing this is that the rpm is not available in Fedora and CentOS and o-a-c is also not in rpm, so.. 

But I can add support for installer in rpm later, currently, only installer next to the o-a-c repo folder works.